Microsoft.Toolkit.Uwp.UI.Animations/ConnectedAnimations/Connected.cs

```diff
@@ -0,0 +1,326 @@
+            return (UIElement)obj.GetValue(AnchorElementProperty);
+        }
+
+        /// <summary>
+        /// Sets the <see cref="UIElement"/> that is the anchor for the coordinated connected animation
+        /// </summary>
+        /// <param name="obj">The <see cref="UIElement"/> that will follow the anchor element</param>
+        /// <param name="value">The <see cref="UIElement"/> that should be followed</param>
+        public static void SetAnchorElement(DependencyObject obj, UIElement value)
+        {
+            obj.SetValue(AnchorElementProperty, value);
+        }
+
+        /// <summary>
+        /// Gets the connected animation key associated with the ListViewBase item being animated
+        /// </summary>
+        /// <param name="obj">The <see cref="Windows.UI.Xaml.Controls.ListViewBase"/></param>
+        /// <returns>The connected animation key</returns>
+        public static string GetListItemKey(DependencyObject obj)
+        {
+            return (string)obj.GetValue(ListItemKeyProperty);
+        }
+
+        /// <summary>
+        /// Sets the connected animation key for the <see cref="Windows.UI.Xaml.Controls.ListViewBase"/> item being animated
+        /// </summary>
+        /// <param name="obj">The <see cref="Windows.UI.Xaml.Controls.ListViewBase"/></param>
+        /// <param name="value">The connected animation key</param>
+        public static void SetListItemKey(DependencyObject obj, string value)
+        {
+            obj.SetValue(ListItemKeyProperty, value);
+        }
+
+        /// <summary>
+        /// Gets the name of the element in the <see cref="DataTemplate"/> that is animated
+        /// </summary>
+        /// <param name="obj">The <see cref="Windows.UI.Xaml.Controls.ListViewBase"/></param>
+        /// <returns>The name of the element being animated</returns>
+        public static string GetListItemElementName(DependencyObject obj)
+        {
+            return (string)obj.GetValue(ListItemElementNameProperty);
+        }
+
+        /// <summary>
+        /// Sets the name of the element in the <see cref="DataTemplate"/> that is animated
+        /// </summary>
+        /// <param name="obj">The <see cref="Windows.UI.Xaml.Controls.ListViewBase"/></param>
+        /// <param name="value">The name of the element to animate</param>
+        public static void SetListItemElementName(DependencyObject obj, string value)
+        {
+            obj.SetValue(ListItemElementNameProperty, value);
+        }
+
+        /// <summary>
+        /// Gets the <see cref="ConnectedAnimationHelper"/> attached to a <see cref="Frame"/>
+        /// Creates a new <see cref="ConnectedAnimationHelper"/> if one is not attached
+        /// </summary>
+        /// <param name="obj">The <see cref="Frame"/></param>
+        /// <returns><see cref="ConnectedAnimationHelper"/> attached to the Frame</returns>
+        private static ConnectedAnimationHelper GetConnectedAnimationHelper(DependencyObject obj)
```

Comment:
Why are you using a DependencyProperty here? They are all Frames, always. And never null. the DP is private. I think the DP should be removed.